cmd/kube-apiserver/app/options/options.go

```diff
@@ -136,9 +135,10 @@ func (s *ServerRunOptions) Flags() (fss cliflag.NamedFlagSets) {
 		"DEPRECATED: kubelet port.")
 	fs.MarkDeprecated("kubelet-port", "kubelet-port is deprecated and will be removed.")
 
-	fs.UintVar(&s.KubeletConfig.ReadOnlyPort, "kubelet-read-only-port", s.KubeletConfig.ReadOnlyPort,
-		"DEPRECATED: kubelet read only port.")
-	fs.MarkDeprecated("kubelet-read-only-port", "kubelet-read-only-port is deprecated and will be removed.")
+	var unused uint
```

Comment:
To simplify and clarify the code, can we consider defining `unused` directly in the `UintVar` function? 
This makes it clear that this variable is only used to discard the flag’s value without adding an extra line of code.

Reply:
Can we initialise the variable like this: 
```go
fs.UintVar(new(uint), "kubelet-read-only-port", 0,
    "DEPRECATED: kubelet read only port. Value ignored even if set.")
```